master/internal/rm/kubernetesrm/pods.go

```diff
@@ -935,12 +935,20 @@ func (p *pods) summarize(ctx *actor.Context) (map[string]model.AgentSummary, err
 		quotaList, err := p.quotaInterfaces[namespace].List(context.TODO(), metaV1.ListOptions{})
```

Comment:
nit: not a big deal. but if we end up with more than just a few namespaces, maybe we'd want to change this so that it doesn't go to the k8s api for every namespace and instead do one lookup for all namespaces and filter on our own.

Reply:
AFAIK the client in question only supports per-namespace lookups in its API. Hope I'm wrong, though. That was the reason for this implementation detail.